media/css/newsletter/newsletter-knowledge-is-power.scss

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-firefox';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-vpn';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/logo-product-pocket';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-mozilla';
+
+.newsletter-knowledge-is-power {
+    .section-subscribe {
+        .mozilla-products {
+            .mozilla-products-list {
```

Comment:
**nit:** nesting creates strong specificity (4x class selector) but we don't seem to be overriding anything that requires strong specificity
<img width="283" alt="Screen Shot 2022-06-14 at 10 49 24 AM" src="https://user-images.githubusercontent.com/19650432/173549351-7fdd8c4c-d6e2-4af2-87f5-d8c7e7b616af.png">
<img width="326" alt="Screen Shot 2022-06-14 at 10 49 56 AM" src="https://user-images.githubusercontent.com/19650432/173549546-9e220420-3ab2-49a5-bf11-aea54804b7e9.png">


**suggestion:** use class names as top level selectors, add specificity only when necessary

Reply:
Nice call out 👍